v1/test/Voting.js

```diff
@@ -0,0 +1,131 @@
+const { didContractThrow } = require("../../common/SolidityTestUtils.js");
+
+const Voting = artifacts.require("Voting");
+
+contract("Voting", function(accounts) {
+  const account1 = accounts[0];
+  const account2 = accounts[1];
+  const account3 = accounts[2];
+  const account4 = accounts[3];
+
+  const getRandomInt = () => {
+    return web3.utils.toBN(web3.utils.randomHex(32));
+  };
+
+  it("One voter, one request", async function() {
+    const voting = await Voting.new();
```

Comment:
I think we should add this contract to the migrations and use `.deployed()`. We can do so in a later PR, but just wanted to note that.

Reply:
I'll do that in a follow up PR.